app/src/main/java/org/simple/clinic/recentpatient/RecentPatientRelativeTimeStampGenerator.kt

```diff
@@ -0,0 +1,34 @@
+package org.simple.clinic.recentpatient
```

Comment:
As discusses [here](https://github.com/simpledotorg/simple-android/pull/639#discussion_r281177666), we do not need instrumented tests for this class, but we should still add a couple of unit tests.

Reply:
done